decidim-core/app/models/decidim/participatory_process_step.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+module Decidim
+  # A ParticipatoryProcess is composed of many steps that hold different
+  # features that will show up in the depending on what step is currently
+  # active.
+  class ParticipatoryProcessStep < ApplicationRecord
+    belongs_to :participatory_process, foreign_key: "decidim_participatory_process_id", class_name: Decidim::ParticipatoryProcess
```

Comment:
Validations of what? I can't validate the translated attributes. Should I validate the dates? (they are optional)